test/auth-request.ts

```diff
@@ -16,15 +16,16 @@
 
 import * as request from 'request';
 
-// TODO: Make the type of `options` more precise
-export const authRequest =
-    (options: (request.UriOptions&request.CoreOptions)|
-     (request.UrlOptions & request.CoreOptions),
-     callback: (err: Error, body: {}, response: request.RequestResponse) =>
-         void) => {
-      request(
-          options,
-          (err: Error, response: request.RequestResponse, body: {}) => {
-            callback(err, body, response);
-          });
-    };
+export async function authRequest(
```

Comment:
Does this need to be an async function? this doesn't seem to use await anywhere in the body.